code/TCP_Srv_Base.h

```diff
@@ -0,0 +1,70 @@
+//  ============================================================
+//  TCP server base class
+//  ============================================================
+//  created by Manfred Sorgo
+
+#pragma once
+#ifndef TCP_SRV_BASE_H
+#define TCP_SRV_BASE_H
+
+#include "BaseTypes.h"
+
+#ifdef _WIN32
+#include <winsock2.h>
+#else
+#include <sys/socket.h>  
+using SOCKET = INT32;
+#endif
+#include <mutex>
+
+#ifndef SELECT_SECONDS
```

Comment:
comment that this can be altered by compiler defines
-> maybe a separate config to illustrate

Reply:
OK, should be alright now